modules/ROOT/pages/sap/sap-connector-xml-maven.adoc

```diff
@@ -0,0 +1,105 @@
+= SAP XML and Maven Support - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../../assets/images/
+
+If you manually code a Mule application in XML either from the Studio XML editor 
+or from a text editor, perform the following actions to get access to the connector:
+
+. Add the namespace for the connector to your XML flow.
+. Add dependency information to your Maven `pom.xml` file.
+
+In the Studio visual editor, when you drag a connector from the palette to the 
```

Comment:
Suggest moving this text to the very beginning, because it sets the stage for when you have to do the same thing manually, which is what the next two tasks are about, right?

So, we are saying if you use the Anypoint Studio visual editor, requirements are automatically met, but if you use an XML or text editor, they are not, and so you have to add the namespace and dependencies? Stating this in this order leads more naturally to the tasks you have to perform in this topic set.